deploy/kubernetes/README.md

```diff
@@ -15,7 +15,7 @@ After that you should be able to visit http://localhost:8080/ui .
 ## Delete mysql DB
 To delete all data written to the DB you can run:
 ```
-kubectl delete -f deploy/kubernetes/db-deployment.yaml
+kubectl delete -k deploy/kubernetes/overlays/gke
 ```
 Alternatively, you use the kubernetes HTTP user interface:
```

Comment:
```suggestion
You can use the kubernetes HTTP user interface to delete specific components, e.g. the MySQL database:
```

Reply:
Does this still work? By default, GKE disables the Kubernetes HTTP UI these days.

Reply:
Ah, it probably does not.